qutip/tests/test_floquet.py

```diff
@@ -67,6 +70,94 @@ def testFloquetUnitary(self):
 
         assert_(max(abs(sol.expect[0] - sol_ref.expect[0])) < 1e-4)
 
+    def testFloquetDissipativeEOps(self):
```

Comment:
These test cases are directly taken from the [docs](http://qutip.org/docs/latest/guide/dynamics/dynamics-floquet.html#floquet-theory-for-dissipative-evolution).